src/screens/credits.rs

```diff
@@ -25,7 +25,7 @@ enum CreditsAction {
     Back,
 }
 
-fn enter_credits(mut commands: Commands) {
+fn show_credits(mut commands: Commands) {
```

Comment:
We should be consistent in naming `show_credits` vs. `show_credits_screen` between the different screens. I'm fine with either.

Reply:
[`f465e65` (#222)](https://github.com/TheBevyFlock/bevy_quickstart/pull/222/commits/f465e65a1fb8f50e06dbdf9a735869ae71f4ef8c)